map/photographer.php

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="de-DE">
+  <head>
+      <title>Bilder eines Fotografen - RailwayStations</title>
+      <?php require_once("./header.php") ?>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["require_once" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/app/RailwayStations/RSWebsite/pullRequest?prid=4676723)